osbs/core.py

```diff
@@ -872,6 +872,23 @@ def restore_resource(self, resource_type, resource):
         check_response(response)
         return response
 
+    def create_config_map(self, config_data):
+        url = self._build_k8s_url("configmaps/")
+        response = self._post(url, data=json.dumps(config_data))
+        check_response(response)
+        return response
+
+    def get_config_map(self, config_name):
+        url = self._build_k8s_url("configmaps/")
+        response = self._get(url, data=json.dumps(config_name))
+        check_response(response)
+        return response
+
+    def delete_config_map(self, config_name):
+        url = self._build_k8s_url("configmaps/")
+        response = self._delete(url, data=json.dumps(config_name))
```

Comment:
The url is incorrect, and the JSON data needs to be formatted correctly (include v1.DeleteOptions): https://docs.openshift.org/latest/rest_api/kubernetes_v1.html#delete-a-configmap